src/openrct2/interface/Colour.h

```diff
@@ -97,115 +96,115 @@ enum
-    PALETTE_INDEX_99 = 99,   //
-    PALETTE_INDEX_102 = 102, // Bright green (lighter)
-    PALETTE_INDEX_108 = 108, //
-    PALETTE_INDEX_111 = 111, //
-    PALETTE_INDEX_129 = 129, // Light Purple (11)
-    PALETTE_INDEX_135 = 135, //
-    PALETTE_INDEX_136 = 136, // Dark Blue (10-11), Light Blue (midlight), Icy Blue (darker)
-    PALETTE_INDEX_138 = 138, // Light Blue (lighter), Icy Blue (middark)
-    PALETTE_INDEX_141 = 141, //
-    PALETTE_INDEX_144 = 144, // Dark Green (1-darkest,
-    PALETTE_INDEX_161 = 161, // Bright Purple (light)
-    PALETTE_INDEX_162 = 162, //
-    PALETTE_INDEX_171 = 171, // Saturated Red (lightest) Bright Red (middark)
-    PALETTE_INDEX_172 = 172, // Saturated Red (10-11), Bright Red (midlight)
-    PALETTE_INDEX_173 = 173, // Used to draw intense lines in the Ride Graphs window
-    PALETTE_INDEX_183 = 183, // Used to draw rides in the Map window
-    PALETTE_INDEX_186 = 186, //
-    PALETTE_INDEX_194 = 194, //
-    PALETTE_INDEX_195 = 195, //
-    PALETTE_INDEX_202 = 202, // Secondary remap 0  / Pink (darkest)
-    PALETTE_INDEX_203 = 203, // Secondary remap 1  / Pink
-    PALETTE_INDEX_204 = 204, // Secondary remap 2  / Pink
-    PALETTE_INDEX_205 = 205, // Secondary remap 3  / Pink
-    PALETTE_INDEX_206 = 206, // Secondary remap 4  / Pink
-    PALETTE_INDEX_207 = 207, // Secondary remap 5  / Pink
-    PALETTE_INDEX_208 = 208, // Secondary remap 6  / Pink
-    PALETTE_INDEX_209 = 209, // Secondary remap 7  / Pink
-    PALETTE_INDEX_210 = 210, // Secondary remap 8  / Pink
-    PALETTE_INDEX_211 = 211, // Secondary remap 9  / Pink
-    PALETTE_INDEX_212 = 212, // Secondary remap 10 / Pink
-    PALETTE_INDEX_213 = 213, // Secondary remap 11 / Pink (lightest)
-    PALETTE_INDEX_222 = 222, //
-    PALETTE_INDEX_230 = 230, // Water (waves)
-    PALETTE_INDEX_231 = 231, // Water (waves)
-    PALETTE_INDEX_232 = 232, // Water (waves)
-    PALETTE_INDEX_233 = 233, // Water (waves)
-    PALETTE_INDEX_234 = 234, // Water (waves)
-    PALETTE_INDEX_235 = 235, // Water (sparkles)
-    PALETTE_INDEX_236 = 236, // Water (sparkles)
-    PALETTE_INDEX_237 = 237, // Water (sparkles)
-    PALETTE_INDEX_238 = 238, // Water (sparkles)
-    PALETTE_INDEX_239 = 239, // Water (sparkles)
-    PALETTE_INDEX_240 = 240, // Track rails
-    PALETTE_INDEX_241 = 241, // Track rails
-    PALETTE_INDEX_242 = 242, // Track rails
-    PALETTE_INDEX_243 = 243, // Primary remap 0
-    PALETTE_INDEX_244 = 244, // Primary remap 1
-    PALETTE_INDEX_245 = 245, // Primary remap 2
-    PALETTE_INDEX_246 = 246, // Primary remap 3
-    PALETTE_INDEX_247 = 247, // Primary remap 4
-    PALETTE_INDEX_248 = 248, // Primary remap 5
-    PALETTE_INDEX_249 = 249, // Primary remap 6
-    PALETTE_INDEX_250 = 250, // Primary remap 7
-    PALETTE_INDEX_251 = 251, // Primary remap 8
-    PALETTE_INDEX_252 = 252, // Primary remap 9
-    PALETTE_INDEX_253 = 253, // Primary remap 10
-    PALETTE_INDEX_254 = 254, // Primary remap 11
+    _0 = 0,     // Transparent
```

Comment:
As unscoped enums bring these into the main namespace I would strongly suggest we keep them with `PaletteIndex` in their name. e.g. `PaletteIndex_0` as otherwise we will potentially cause collisions with any other thing we name _0. (Although i do get you are doing this as a first step so maybe that would be a pain)

Reply:
I think the better solution would be to just put it under the namespace PaletteIndex it and make the enum anonymous, that at least wont ever clash with any other symbol.

Reply:
I have now changed the names to `PaletteIndex::pi0`. Having them as a weak enum is only meant as a temporary measure anyway, the goal is to refactor the code to make them an enum class.

Reply:
>  If most things use the correct type

If they did, I would have made it `enum class` to begin with.

Reply:
What I meant is that if you change some of the arguments to PaletteIndex it won't require much of enumValue usage other than few memsets from what I'm seeing so why not just do that in one go?

Reply:
I have _tried_ that. If that had been feasible, I would have done that, just like I did in all other PRs where I refactored weak enums.